lib/compare_links.rb

```diff
@@ -38,9 +38,15 @@ def api_url_method
 
   def sort(links)
     links.each_with_object({}) do |(k, v), h|
-      h[k] = v.map do |value|
+      v = v.map do |value|
```

Comment:
this is fine, although feel slightly apprehensive about clobbering the passed in variable `v`